### ISSUE TITLE
Could not parse package.json

### ISSUE THREAD

**Issue by sumanthratna** [pastelsky/bundlephobia#372]
**Describe the bug**
When I upload my `package.json` via both drag-and-drop and click-to-upload, I get an alert saying "Could not parse package.json"

See #64, #5, and #89. 

**To Reproduce**
Here's my package.json:
```json
{
  "name": "hacktj-2021",
  "version": "1.0.0",
  "private": true,
  "homepage": "https://hacktj.org/2021",
  "dependencies": {
    "@types/chart.js": "^2.9.22",
    "@types/jest": "^26.0.3",
    "@types/lodash": "^4.14.157",
    "@types/node": "^14.0.14",
    "@types/react": "^16.9.41",
    "@types/react-dom": "^16.9.8",
    "@types/react-map-gl": "^5.2.4",
    "@types/reactstrap": "^8.4.3",
    "@typescript-eslint/parser": "^3.0.0",
    "chart.js": "^3.0.0-alpha",
    "history": "^5.0.0",
    "lodash": "^4.17.15",
    "mapbox-gl": "^1.11.0",
    "react": "^16.13.1",
    "react-chartjs-2": "^2.9.0",
    "react-dom": "^16.13.1",
    "react-map-gl": "^5.2.7",
    "react-parallax-tilt": "^1.4.56",
    "react-router": "^6.0.0-beta.0",
    "react-router-dom": "^6.0.0-beta.0",
    "react-scripts": "^3.4.1",
    "react-spinners": "^0.9.0",
    "reactstrap": "^8.4.1",
    "source-map-explorer": "^2.4.2",
    "typescript": "^3.9.6",
    "web-vitals": "^0.2.3"
  },
  "scripts": {
    "analyze": "source-map-explorer 'build/static/js/*.js'",
    "start": "yarn run switch-homepage && react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject",
    "format": "prettier src/ --write",
    "lint": "eslint src/ --fix",
    "tint": "./tint-sponsors '#6BE667'",
    "switch-event": "./switch event",
    "switch-homepage": "./switch homepage",
    "predeploy": "yarn run build && NODE_DEBUG=gh-pages",
    "deploy-event": "yarn run switch-event && yarn run predeploy && timestamp=$(date -u +\"%FT%TZ\") && gh-pages -d build -m \"Update $timestamp\"",
    "deploy-homepage": "yarn run switch-homepage && yarn run predeploy && timestamp=$(date -u +\"%FT%TZ\") && gh-pages -b master -d build -m \"Update $timestamp\" && yarn run switch-event",
    "deploy-all": "yarn run deploy-event && yarn run deploy-homepage"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "@typescript-eslint/eslint-plugin": "latest",
    "eslint": "^7.4.0",
    "eslint-config-airbnb": "18.2.0",
    "eslint-config-airbnb-typescript-prettier": "^3.1.0",
    "eslint-import-resolver-node": "^0.3.4",
    "eslint-plugin-import": "^2.21.2",
    "eslint-plugin-jsx-a11y": "^6.3.1",
    "eslint-plugin-prettier": "latest",
    "eslint-plugin-react": "^7.20.0",
    "eslint-plugin-react-hooks": "^4.0.4",
    "gh-pages": "^3.1.0",
    "prettier": "^2.0.5"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/HackTJ/2021.git"
  },
  "bugs": {
    "url": "https://github.com/HackTJ/2021/issues"
  }
}
```

**Expected behavior**
It parses my JSON correctly.

**Comment by vasilii-kovalev** [comment]
The same for such `package.json`:
```json
{
  "name": "hydrate-text",
  "version": "1.1.5",
  "description": "Tiny library for dynamic text hydrating with variables",
  "main": "lib/index",
  "module": "lib/index",
  "types": "lib/index.d.ts",
  "sideEffects": false,
  "keywords": [
    "javascript",
    "typescript",
    "template",
    "string",
    "formatter",
    "zero-dependency",
    "tree-shakable"
  ],
  "author": "Vasilii Kovalev <vasya.kowalev@yandex.ru>",
  "license": "MIT",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/vasilii-kovalev/hydrate-text.git"
  },
  "homepage": "https://github.com/vasilii-kovalev/hydrate-text#readme",
  "bugs": {
    "url": "https://github.com/vasilii-kovalev/hydrate-text/issues"
  },
  "devDependencies": {
    "@types/jest": "^26.0.5",
    "@typescript-eslint/eslint-plugin": "^3.7.0",
    "@typescript-eslint/parser": "^3.7.0",
    "eslint": "^7.5.0",
    "eslint-config-airbnb-base": "^14.2.0",
    "eslint-plugin-import": "^2.22.0",
    "husky": "^4.2.5",
    "jest": "^26.1.0",
    "ts-jest": "^26.1.3",
    "typescript": "^3.9.7"
  },
  "scripts": {
    "test": "jest",
    "tslint": "tsc --noEmit",
    "eslint": "eslint --ext .ts src --color",
    "build": "tsc",
    "prepare": "npm run build",
    "prepublishOnly": "npm run tslint && npm run eslint && npm test",
    "postversion": "git push && git push --tags"
  },
  "jest": {
    "preset": "ts-jest",
    "testEnvironment": "node",
    "collectCoverage": true,
    "collectCoverageFrom": [
      "src/index.ts"
    ],
    "coverageDirectory": "./coverage"
  },
  "husky": {
    "hooks": {
      "pre-commit": "npm run prepublishOnly"
    }
  }
}
```

**Comment by vasilii-kovalev** [comment]
@sumanthratna, it seems it is working now:
![image](https://user-images.githubusercontent.com/10310491/88448792-dd5cc380-ce49-11ea-942f-0eb5df5f92cd.png)

I've investigated the problem by myself. I created a `package.json` file with some fields (via `npm init -y`) and tried to parse it via the ["Scan package.json"](https://bundlephobia.com/scan) - it fails. Than I added a dependency to `dependencies` section of the JSON file and it was parsed OK. Then I added `dependencies` section to my version of `package.json` and it was parsed OK as well.

On the one hand, it is correct that if there is no dependencies there shouldn't be any dependencies analysis performed, but on the other hand, it would be great to see more informative message, __why exactly__ the parsing fails.
